src/js/apps/globals/app-frame_app.js

```diff
@@ -9,10 +11,56 @@ import PatientsMainApp from 'js/apps/patients/patients-main_app';
 
 import { AppNavView, AppNavCollectionView } from 'js/views/globals/app-nav/app-nav_views';
 
+const patientsNav = new Backbone.Collection([{
+  titleI18nKey: 'globals.appNav.patients.allPatients',
+  event: 'patients:all',
+  eventArgs: [],
+}]);
+
+const viewsNav = new Backbone.Collection([
+  {
+    titleI18nKey: 'globals.appNav.views.ownedByMe',
+    event: 'view',
+    eventArgs: ['owned-by-me'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.myRole',
+    event: 'view',
+    eventArgs: ['actions-for-my-role'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.newActions',
+    event: 'view',
+    eventArgs: ['new-actions'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.updatedPastThree',
+    event: 'view',
+    eventArgs: ['updated-past-three-days'],
+  },
+  {
+    titleI18nKey: 'globals.appNav.views.doneLastThirty',
+    event: 'view',
+    eventArgs: ['done-last-thirty-days'],
+  },
+]);
+
 export default App.extend({
   channelName: 'nav',
   radioRequests: {
-    'select': 'select',
+    'select': 'onSelect',
+  },
+  onSelect(appName, event, eventArgs) {
+    const navMatch = viewsNav.find(model => this._navMatch(model, event, eventArgs))
+      || patientsNav.find(model => this._navMatch(model, event, eventArgs));
+
+    if (navMatch) {
+      this.getChildView('nav').$('.is-selected').removeClass('is-selected');
```

Comment:
as much as it seems unnecessary I'd like to move this to a function on the nav view so that  DOM querying isn't in the apps

Reply:
Ah there is a `removeSelected()` just not being used